tests/10-deploy-test

```diff
@@ -63,5 +63,22 @@ class TestDeployment(unittest.TestCase):
             message = 'Could not delete the busybox container.'
             amulet.raise_status(amulet.FAIL, msg=message)
 
+    def test_latest_config_option(self):
```

Comment:
I'm wondering if we shouldn't define a style guide for the tests so we maintain a consistent feel and language for them. Might be a nice candidate for the docs.